aztec-up/bin/aztec-nargo

```diff
@@ -5,7 +5,9 @@ export SKIP_NET=1
 export PATH=$PATH:$HOME/bin
 
 if [ "${1:-}" == "lsp" ]; then
-  docker run -i -v $HOME:$HOME -e HOME=$HOME aztecprotocol/noir $@
+	docker run -i -v $HOME:$HOME -e HOME=$HOME aztecprotocol/noir $@
+elif [ "${1:-}" == "compile" ]; then
```

Comment:
I'd rather not introduce another binary / container to release and leak the internal composition to the outside world.
Can we instead, get rid of noir container, avm-transpiler container, and just have an `aztecprotocol/aztec-nargo` container which contains both nargo, transpiler, and it internally, inside the container, performs this wrapping modification to the `compile` command?